src/function.ts

```diff
@@ -0,0 +1,3 @@
+export default function helloWorld() {
```

Comment:
Just a function to test that running `npm run test` works with Typescript files. Can be deleted.

Reply:
Fine with this but can you just inline it directly into the test and call it helloWorld.u.spec.ts?